packages/app-portal/src/systems/Ecosystem/data/projects.json

```
@@ -144,8 +166,8 @@
   },
   {
     "isLive": true,
-    "name": "Fuel-Bridge",
-    "url": "https://alpha.fuel.network/bridge",
+    "name": "Fuel Bridge",
+    "url": "https://app.fuel.network/portal/bridge",
```

we've removed this `/portal` 

```suggestion
    "url": "https://app.fuel.network/bridge",
```